test/unit/spec/MfaVerify_spec.js

```diff
@@ -174,6 +174,39 @@ function (Okta,
       });
     }
 
+    function setupWebauthnOnly() {
+      var setNextResponse = Util.mockAjax();
+      var baseUrl = 'https://foo.com';
+      var authClient = new OktaAuth({url: baseUrl, transformErrorXHR: LoginUtil.transformErrorXHR});
+      var successSpy = jasmine.createSpy('success');
+      var router = new Router(_.extend({
```

Comment:
I think creating another function here may cause a lint error similar to my first commit.

`/home/travis/build/okta/okta-signin-widget/test/unit/spec/MfaVerify_spec.js
  114:33  error  Function has too many statements (35). Maximum allowed is 34  max-statements`
 
Should I create another describe? Is there a better way around it?